packages/generators/app/src/create-quickstart-project.ts

```diff
@@ -14,7 +14,7 @@ export default async function createQuickStartProject(scope: Scope) {
   const configuration: Configuration = {
     client,
     connection: defaultConfigs[client],
-    dependencies: clientDependencies({ client }),
+    dependencies: { ...clientDependencies({ client }), ...scope.additionalsDependencies },
```

Comment:
just curious, where are the additional dependencies loaded from?

Reply:
They're defined in the `scope` which in `generateNew` includes `react` etc.